src/script/properties/PropertiesRepository.ts

```diff
@@ -292,15 +320,7 @@ class PropertiesRepository {
     }
   }
 
-  /**
-   * Set the preference of specified type
-   *
-   * @private
-   * @param {PROPERTIES_TYPE} propertiesType - Type of preference to set
-   * @param {*} changedPreference - New preference to set
-   * @returns {undefined} No return value
-   */
-  _setPreference(propertiesType, changedPreference) {
+  _setPreference(propertiesType: string, changedPreference: {}): void {
```

Comment:
```suggestion
  private _setPreference(propertiesType: string, changedPreference: {}): void {
```

Reply:
Good reminder. I will also remove the leading underscore then.